app/controllers/schools/advice/storage_heaters_controller.rb

```diff
@@ -6,12 +6,12 @@ class StorageHeatersController < AdviceBaseController
       before_action :set_annual_usage_breakdown, only: [:insights, :analysis]
       before_action :set_usage_categories, only: [:insights, :analysis]
       before_action :set_heating_thermostatic_analysis, only: [:insights, :analysis]
+      before_action :set_no_how_we_analysed_modal, only: [:insights]
+      before_action :set_analysis_dates, only: [:analysis]
```

Comment:
I think its a bug that the modal links aren't shown on the storage heater pages.

Reply:
added links to storage heater pages